tox.ini

```diff
@@ -164,6 +169,10 @@ commands =
 
 [testenv:pylint-examples]
 basepython = {[testenv:pylint]basepython}
+# This does not actually ignore errors,
+#  it just runs all commands regardless of whether any fail.
+#  If any fail, the final result is still a fail.
```

Comment:
np: extra whitespace?

np: `the final result is still a fail` -> `the final result will still fail`